src/main/g8/src/main/scala/com/lightbend/akka/http/sample/JsonSupport.scala

```diff
@@ -0,0 +1,14 @@
+package com.lightbend.akka.http.sample
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import com.lightbend.akka.http.sample.UserRegistryActor.ActionPerformed
+import spray.json.{ DefaultJsonProtocol, RootJsonFormat }
+
+trait JsonSupport extends SprayJsonSupport {
```

Comment:
Esp for support traits that's how we tell people to do stuff yeah. Wanted to be consistent